dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/bundle/ReportSummaryIntegrationTest.java

```diff
@@ -93,28 +91,6 @@ void testStatsCountForOneCreatedAndOneUpdatedTE() throws IOException {
     assertEquals(0, trackerImportTeReport.getStats().getDeleted());
   }
 
-  @Test
-  void testStatsCountForOneCreatedAndOneUpdatedTEAndOneInvalidTE() throws IOException {
```

Comment:
Why is that not needed anymore?

Reply:
We cannot have an invalid UID at this point anymore.
We will have some web tests to check this when we will change the view model

Reply:
This test was not about an invalid UID. It just used an invalid UID to test the report. So this should not be removed but the TE made invalid in some other way.

Reply:
You are right :sweat_smile: 
Putting it back also for other entities